src/components/ObsDetails/DetailsTab.js

```diff
@@ -0,0 +1,157 @@
+          </View>
+          <Divider />
+        </>
+      )}
+
+      <View className="flex-row justify-between items-center mt-[8px] mx-[15px]">
+        <Heading4>{t( "LOCATION" )}</Heading4>
+        <KebabMenu
+          visible={locationKebabMenuVisible}
+          setVisible={setLocationKebabMenuVisible}
+        >
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Share-location" )}
+          />
+          <Menu.Item
+            onPress={async ( ) => {
+
+            }}
+            title={t( "Copy-coordinates" )}
+          />
+        </KebabMenu>
+      </View>
+      {isOnline
+        ? (
+          <Map
+            obsLatitude={observation.latitude}
+            obsLongitude={observation.longitude}
+            mapHeight={230}
+          />
+        ) : (
+          <View className="h-16 items-center justify-center">
+            <IconMaterial
+              name="wifi-off"
+              size={30}
+              accessibilityRole="image"
+              accessibilityLabel={t( "Location-map-unavailable-without-internet" )}
+            />
+          </View>
+        )}
+      <View className={`mt-[11px] ${sectionClass}`}>
+        <ObservationLocation observation={observation} details />
+      </View>
+      <Divider />
+
+      <View className={sectionClass}>
+        <Heading4 className={headingClass}>{t( "DATE" )}</Heading4>
+        <DateDisplay
+          classNameMargin="mb-[12px]"
+          label={t( "Observed" )}
+          dateString={checkCamelAndSnakeCase( observation, "timeObservedAt" )}
+        />
+        <DateDisplay
+          label={t( "Uploaded" )}
+          dateString={checkCamelAndSnakeCase( observation, "createdAt" )}
+        />
+      </View>
+      <Divider />
+
+      <View className={`${sectionClass} flex-col`}>
+        <Heading4 className={headingClass}>{t( "DATA-QUALITY" )}</Heading4>
+        <View className="space-y-[15px]">
+          <View className="flex-row justify-around">
+            {displayQualityGradeOption( "casual" )}
+            {displayQualityGradeOption( "needs_id" )}
+            {displayQualityGradeOption( "research" )}
+          </View>
+          <Body4>
+            {t( "This observation needs more identifications to reach research grade" )}
+          </Body4>
+          <Button text={t( "VIEW-DATA-QUALITY-ASSESSEMENT" )} />
+        </View>
+      </View>
+      <Divider />
+
+      <View className={sectionClass}>
+        <Heading4 className={headingClass}>{t( "PROJECTS" )}</Heading4>
+        <Button text={t( "VIEW-PROJECTS" )} />
+      </View>
+
+      <Divider />
+
+      <View className={`${sectionClass} space-y-[11px]`}>
+        <Heading4 className={headingClass}>{t( "OTHER-DATA" )}</Heading4>
+        <Attribution observation={observation} />
+        {application && (
+          <View className="flex-row">
+            <Body4>{t( "Uploaded-via" )}</Body4>
+            <Body4>{` ${application}`}</Body4>
```

Comment:
These two things need to be one string, like `t( "Uploaded_via_application", { application } )` because translators need control over word ordering (e.g. for RTL languages).